eradiate/scenes/atmosphere/_core.py

```diff
@@ -71,66 +59,48 @@ class Atmosphere(SceneElement, ABC):
         default='"auto"',
     )
 
-    def height(self):
-        """
-        Actual value of the atmosphere's height as a :class:`pint.Quantity`.
-        If ``toa_altitude`` is set to ``"auto"``, a value of 100 km is returned;
-        otherwise, ``toa_altitude`` is returned.
-        """
-        if self.toa_altitude == "auto":
-            return ureg.Quantity(100.0, ureg.km)
-        else:
-            return self.toa_altitude
+    # --------------------------------------------------------------------------
+    #                       Abstract instance methods
+    # --------------------------------------------------------------------------
```

Comment:
Rather than whether methods are abstract or not, I'd prefer to categorise components based on their intent as part of the interface this class defines. In that case, relevant categories would be:

- fields and init sequence (no banner needed)
- properties (characterise the object itself)
- kernel dictionary generation